apps/web/src/components/EmployeeSearchInput/EmployeeSearchInput.tsx

```diff
@@ -0,0 +1,79 @@
+import { useFormContext, Controller } from "react-hook-form";
+import { ComponentPropsWithoutRef } from "react";
+
+import {
+  CommonInputProps,
+  Field,
+  useFieldState,
+} from "@gc-digital-talent/forms";
+import { HydrogenAttributes } from "@gc-digital-talent/ui";
+import { workEmailDomainRegex } from "@gc-digital-talent/helpers";
+
+import ControlledInput from "./ControlledInput";
+import { EmployeeSearchValue, ErrorMessages } from "./types";
+export type { EmployeeSearchValue, ErrorMessages } from "./types";
+
+interface WrapperProps
+  extends ComponentPropsWithoutRef<"div">,
+    HydrogenAttributes {}
+
+export interface EmployeeSearchInputProps
+  extends Omit<ComponentPropsWithoutRef<"input">, "id" | "name">,
+    Omit<CommonInputProps, "context"> {
+  buttonLabel?: string;
+  wrapperProps: WrapperProps;
```

Comment:
Is this a mandatory prop?  It doesn't appear in your example.  It doesn't seem like it should be mandatory.